src/genomicsdb.pxd

```diff
@@ -111,20 +111,20 @@ cdef extern from "genomicsdb.h":
         GenomicsDBVariants query_variants()
 
         # query_variant_calls(const std::string& array, genomicsdb_ranges_t column_ranges=SCAN_FULL, genomicsdb_ranges_t row_ranges={});
-        GenomicsDBVariantCalls query_variant_calls(string, genomicsdb_ranges_t, genomicsdb_ranges_t) except +
-        GenomicsDBVariantCalls query_variant_calls(string, genomicsdb_ranges_t) except +
-        GenomicsDBVariantCalls query_variant_calls(string) except +
+        GenomicsDBVariantCalls query_variant_calls(string, genomicsdb_ranges_t, genomicsdb_ranges_t) except + nogil
```

Comment:
Do we need these explicit `nogil` attributes here since we are using `with nogil:` while using query_variant_calls in genomicsdb.pyx?

Reply:
Not sure if we need it -- but just following what I read here: https://cython.readthedocs.io/en/latest/src/userguide/nogil.html#marking-functions-as-able-to-run-without-the-gil

Reply:
Read the docs a bit more, and I think I do need these here. Otherwise, during build time I get a complaint that I'm trying to call a gil-requiring function without gil